exasol/nb_connector/text_ai_extension_wrapper.py

```diff
@@ -122,4 +149,53 @@ def initialize_text_ai_extension(conf: Secrets,
         allow_override:
             If True allows overriding the language definition.
     """
-    pass
+
+    # Make the connection object names
+    db_user = str(conf.get(CKey.db_user))
+    bfs_conn_name = "_".join([BFS_CONNECTION_PREFIX, db_user])
+    #container_name = TXAIELanguageContainerDeployer.SLC_NAME,#todo needs release or hardcode for now
+    container_name = "exasol_text_ai_extension_container_release.tar.gz"
+
+
+    if run_deploy_container:
+        if version:
+            install_txaie_version(version)
+        elif container_file:
```

Comment:
because, we store the zip location in the ai-lab config, I would suggest, that we also store the container file there after we unpacked it from the zip. Reason: @ahsimb assumes in his ai-lab PR that only the ai-lab config goes into initialize_text_ai_extension. https://github.com/exasol/ai-lab/pull/356/files#diff-cc453567745e1feb88fcfe9fda736fc494590009465549c1d0bec4451a9c7552

Reply:
changed